gradle/versions.gradle

```diff
@@ -29,6 +29,8 @@ dependencyManagement {
 
     dependency 'com.moandjiezana.toml:toml4j:0.7.2'
 
+    // dependency 'com.oracle:ojdbc8:18.3.0.0'
```

Comment:
Don't need to comment this out

Reply:
Good catch. Fixed!

Reply:
Can we actually reference this as part of our buildsystem? Or does the dependency need to be found via cmdline?